### PR TITLE
Display Standard and Premium support level for FR customers

### PR DESCRIPTION
**Pull request by marie-j**
## Display Standard and Premium support level for FR customers

ref: MANAGER-3926

⚠️ Needs sync, do not prod until 19th dec

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/account/user/dashboard/user-dashboard.routing.js [hunk 1]
```diff
@@ -4,34 +4,29 @@ import {
   USER_DASHBOARD_SHORTCUTS,
 } from './user-dashboard.constants';
 
-import component from './user-dasboard.component';
-import './user-dashboard.less';
+export default /* @ngInject */ ($stateProvider) => {
+  const name = 'app.account.user.dashboard';
```
Comment (antleblanc) [inline packages/manager/apps/dedicated/client/app/account/user/dashboard/user-dashboard.routing.js]: Declare `const` is not needed in your routing file.

File: packages/manager/apps/dedicated/client/app/account/user/dashboard/user-dahboard.module.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+const moduleName = 'ovhManagerDedicatedUserAccountDashboard';
+
+angular
+  .module(moduleName, [
+    'ui.router',
+  ])
+  .config(routing)
+  .component('userAccountDashboard', component);
+
+
```
Comment (antleblanc) [inline packages/manager/apps/dedicated/client/app/account/user/dashboard/user-dahboard.module.js]: Extra empty line:)

**Comment by marie-j** [comment]
Quality check is ok but we need sync, should be prodded the 19th dec

**Comment by JeremyDec** [comment]
@marie-j there is conflict

**Comment by marie-j** [comment]
> @marie-j there is conflict

@JeremyDec conflicts solved

File: packages/manager/apps/dedicated/client/app/account/user/dashboard/user-dahboard.module.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import component from './user-dasboard.component';
+import routing from './user-dashboard.routing';
+
+import './user-dashboard.less';
+
+const moduleName = 'ovhManagerDedicatedUserAccountDashboard';
+
+angular
+  .module(moduleName, [
+    'ui.router',
```
Comment (FredericEspiau) [inline packages/manager/apps/dedicated/client/app/account/user/dashboard/user-dahboard.module.js]: `oui` and `pascalprecht.translate` ?

**Pull request merged**